specificatie/Waardelijsten/openapi.yaml

```diff
@@ -110,16 +110,17 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/responses/501"
         '503':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/responses/503"
-      tags: 
+      tags:
         - Waardelijsten
   /waardelijsten/{waardelijstidentificatie}/waarden:
     get:
       operationId: getWaarden
-      description: "Het ophalen van de waarden van een waardelijst <br> Als er op waarde of code gezocht wordt worden alleen actuele waarden geretourneerd. <br> Als de parameter <b> inclusief beeindigd </b> met waarde <i> true </i>  wordt meegegeven bij de call worden ook beeindigde waarden geretourneerd. <br> Bij het zoeken op de omschrijving mag er gebruik worden gemaakt van [wildcards](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/wildcard.feature). Zoeken is **case-insensitive**."
-      parameters: 
+      description: "Het ophalen van de waarden van een waardelijst.<br>Default worden alleen actuele waarden geretourneerd. Met de parameter <b>inclusiefbeeindigd</b> kunnen ook beëindigde waarden gevraagd."
```

Comment:
Met de parameter <b>inclusiefbeeindigd</b> kunnen ook beëindigde waarden gevraagd. =>
Met de parameter <b>inclusiefbeeindigd</b> kunnen ook beëindigde waarden worden gevraagd.

Reply:
aangepast